packages/rx_bloc_cli/mason_templates/bricks/feature_qr_scanner/__brick__/lib/l10n/network_lookup/qr_code_lookup.dart

```diff
@@ -1,15 +1,17 @@
 {{> licence.dart }}
 
 import '../../assets.dart';
-import '../{{project_name}}_app_i18n.dart';
-import 'util.dart';
+import '../{{project_name}}_app_i18n.dart';{{#enable_remote_translations}}
```

Comment:
My understanding is that the network_loopkup files are not needed in case the remote translations are not enabled.